jetty-maven-plugin/src/it/javax-annotation-api/pom.xml

```diff
@@ -45,9 +45,9 @@
       <version>1.8.0-beta2</version>
     </dependency>
     <dependency>
-      <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-simple</artifactId>
-      <version>1.8.0-beta2</version>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j18-impl</artifactId>
+      <version>2.11.1</version>
```

Comment:
Do we really need the version here?
I would prefer it is inherited from somewhere, so that we don't forget to update it to newer versions.

Reply:
easy to fix.
I have a bit more concern about the file ` jetty-util/src/main/config/modules/slf4j-api.mod ` 
and the value ` slf4j.version?=1.8.0-beta2 `  (without the change included in the pr this was not working anymore)